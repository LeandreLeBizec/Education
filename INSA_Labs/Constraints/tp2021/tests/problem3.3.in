problem3.
23. 99999999.
