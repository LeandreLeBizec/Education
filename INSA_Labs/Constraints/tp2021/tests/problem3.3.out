[15, 12, 7, 21, 21, 1]
